paymentapi/src/main/java/fi/hel/verkkokauppa/payment/logic/util/PaymentUtil.java

```diff
@@ -1,6 +1,7 @@
-package fi.hel.verkkokauppa.payment.logic;
+package fi.hel.verkkokauppa.payment.logic.util;
```

Comment:
As the util is not actually any "logic" could be also payment.util instead of payment.logic.util

Reply:
I agree with this, I will move this class to   payment.util